src/content/en/updates/2017/04/native-mac-os-notifications.md

```diff
@@ -0,0 +1,145 @@
+{# wf_tags: chrome59,notifications #}
+{# wf_featured_image: /web/updates/images/generic/notifications.png #}
+{# wf_featured_snippet: As of Chrome version 59, notifications will be displayed native notifications on Mac OS X. #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# Moving to the the Native Notification System on Mac OS X {: .page-title }
+
+Starting in Chrome 59, notifications sent via the [Notifications
+API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) or the
+[chrome.notifications extensions
+API](https://developer.chrome.com/extensions/notifications) will be shown
+directly by the Mac OS X native notification system instead of Chrome's own system.
+
+This change makes Chrome on Mac OS X feel much better integrated into the platform
+and fixes a number of long standing bugs, such as Chrome not respecting the
+system Do Not Disturb setting.
+
+Below we'll look at the differences this change introduces to the existing
+API's.
+
+## Notification Center
+
+One of the benefits (and feature requests from users) of this change is that
+notifications will be displayed in OS X's notification center.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image00.png">
+  <figcaption>
+    Google Chrome Notifications will be displayed in the Mac OS X notification center.
+  </figcaption>
+</figure>
+
+## Differences
+### Icon Size and Positioning
+
+The appearance of icons will change. They'll be smaller in size and padding is
+applied. You may want to consider switching to a transparent background icon
+instead of a solid color to be aesthetically pleasing.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image01.png">
+  <figcaption>
+    Before and after of notification icons displayed by Chrome vs displayed by
+    Mac OS X.
+  </figcaption>
+</figure>
+
+### Action Icons
+
+Before this change action buttons and icons would be displayed in the
+notification. With the native notifications the action button icons will not be
+used and the user will need to hover over the notification and select the "More"
+button to see the available actions.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image02.png">
+  <figcaption>
+    Before and after of notification action buttons with icons displayed by
+    Chrome vs displayed by Mac OS X.
+  </figcaption>
+</figure>
+
+### Chrome Logo
+
+The Chrome logo will always be displayed and cannot be replaced or altered. This
+is a requirement for third party applications on Mac OS X.
+
+### Images
+
+The image option will no longer be supported on OS X. If you define an image
+property the notification will still be displayed, but it will ignore the image
+parameter (See example below).
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image03.png">
+  <figcaption>
+    Before and after of notification image displayed by Chrome vs displayed by Mac OS X.
```

Comment:
You could just simplify to 'Before and after for Chrome on Mac.'  Everything I've removed to get that is provided by the context. Regardless, the old sentence is bit of a word salad and should be revised.